### PR TITLE
LIVY-275. Add livy REST API to detect Livy version info

### PR DESCRIPTION
**Pull request by jerryshao**

### CODE REVIEW

**Comment by jerryshao** [comment]
\CC @zjffdu

**Comment by zjffdu** [comment]
LGTM

**Comment by zjffdu** [comment]
ping @felixcheung

File: dev/livy-build-info [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```
Comment (felixcheung) [inline dev/livy-build-info]: should this file be livy-build-info`.sh`?

**Comment by felixcheung** [comment]
Nice!

File: dev/livy-build-info (hunk 1 above)
Comment (jerryshao) [inline dev/livy-build-info]: Sure, I will rename it.

**Pull request merged**